config/reports/links-between-worship-services-and-admin-units-report.js

```diff
@@ -71,7 +76,7 @@ export default {
         adminUnit: unit.adminUnit.value,
```

Comment:
This is still missing the relevant lines `worshipAdministrativeUnitRelationship` and `'worshipAdministrativeUnitRelationshipLabel`.